frontend/src/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const BACKEND_API_URL =
+  process.env.BACKEND_API_URL || "http://localhost:42069";
+export const SIGN_OUT_CALLBACK_URL = "/sign-in";
+export const SIGN_IN_CALLBACK_URL = "/";
```

Comment:
mover o sign in url para o path `/dashboard`, porque iremos ter outras páginas que se montarão na raiz do projeto como a própria home, pricing, about, status, comunidades

Reply:
resolvido